index.js

```diff
@@ -244,6 +246,35 @@ class GithubScm extends Scm {
             }
         };
     }
+
+    /**
+     * Return a unique identifier for the scmUrl
+     * @method getId
+     * @param {Object}    config        Configuration
+     * @param {String}    config.scmUrl The scmUrl to generate ID
+     * @param {String}    config.token  The token used to authenticate to the SCM
+     * @return {Promise}
+     */
+    getId(config) {
```

Comment:
Oops I accidentally approved this PR, but this needs to be addressed before merging.